src/controllers/user.controller.js

```diff
@@ -1 +1,32 @@
-exports.status = async (req, res, next) => res.json({ message: 'User endpoint is working.' })
\ No newline at end of file
+const User = require('../models/user.model')
+const errorHandler = require('../middlewares/error-handler')
+
+exports.status = async (req, res, next) => res.json({ message: 'User endpoint is working.' })
+
+exports.add_user = async (req, res, next) => {
+  const { title, username, firstname, lastname } = req.body;
+  const newUser = {
+    profile_picture:'update.to.s3.default',
+    title,
+    badges: 'General',
```

Comment:
Shouldn't badges be an array?

Reply:
I'd like to think of badges as collectables where they can have a variety of badges from events, but can also equip a badge a time to showcase. Think of MAPLESTORY medals :^)